recipes/libuwebsockets/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "libuwebsockets" %}
+{% set version = "20.36.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/uNetworking/uWebSockets/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: cda266f7ed6abe67ef3cae6e223a580fe5091db9156c1f4123ee328ae21511c9
+
+build:
+  number: 0
+  # uwebsockets is a header-only library that uses usockets and zlib, so any user of it eventually links libusockets
+  run_exports:
+    - {{ pin_compatible('libusockets', max_pin='x.x.x') }}
+    - {{ pin_compatible ('libzlib', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkg-config
+    - ninja
+  host:
+    - libusockets
+    - zlib
+    # This is just to provide FinduSockets.cmake, not needed at runtime
+    - ycm-cmake-modules
```

Comment:
Then it belongs in the build section? Or that doesn't work because it gets confused.

Reply:
This is a good point, it is indeed a build dependency, I fixed it in the feedstock https://github.com/conda-forge/libuwebsockets-feedstock/pull/1 .